tests/repo/open.c

```diff
@@ -35,7 +35,53 @@ void test_repo_open__format_version_1(void)
 
 	git_config_free(config);
 	git_repository_free(repo);
+
+	git_repository_open(&repo, "empty_bare.git");
+	cl_assert(git_repository_path(repo) != NULL);
+	cl_assert(git__suffixcmp(git_repository_path(repo), "/") == 0);
+	git_repository_free(repo);
```

Comment:
I don't understand this addition, can you explain it in a comment on the commit?

Reply:
This is just the way the tests above check for success.  If you're happy with just checking for success opening the repository, I'm happy to just do that.

Reply:
Huh!  OK, there's probably a reason for this, but I've forgotten what it is.  😀 Let's keep it.